spring-integration-jdbc/src/main/java/org/springframework/integration/jdbc/ExpressionEvaluatingSqlParameterSourceFactory.java

```diff
@@ -126,9 +129,23 @@ public void setParameterExpressions(Map<String, String> parameterExpressions) {
 		this.parameterExpressions = paramExpressions;
 	}
 
+	/**
+	 * Optionally set parameter types for the used parameters. Use
+	 * {@link java.sql.Types} to get the parameter type value.
+	 * @param sqlParametersTypes the parameter type to set
+	 * @see java.sql.Types
+	 * @since 5.0
```

Comment:
Will fix.

Reply:
I would say no reason to comment if you are agreed with me. Otherwise I have this extra notification in my email box almost for nothing. 

Having your silent fix is enough for me to determine that you've been agreed with my concern.

Thanks for understanding.